src/gcp/jwt.rs

```diff
@@ -114,7 +114,7 @@ impl Default for Algorithm {
     }
 }
 
-/// The supported RSA key formats, see the documentation for ring::signature::RsaKeyPair
+/// The supported RSA key formats, see the documentation for `ring::signature::RsaKeyPair`
```

Comment:
```suggestion
/// The supported RSA key formats, see the documentation for [`ring::signature::RsaKeyPair`]
```
Now that intradoc links are stable and working, prefer to use those whenever we reference types.